crates/oxc_linter/src/snapshots/no_import_type_side_effects.snap

```diff
@@ -0,0 +1,27 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: no_import_type_side_effects
+---
+  ⚠ typescript-eslint(no-import-type-side-effects): TypeScript will only remove the inline type specifiers which will leave behind a side effect import at runtime. Convert this to a top-level type qualifier to properly remove the entire import.
```

Comment:
"Convert this to a top-level type qualifier to properly remove the entire import." should be in the help section, along with an example.